rust/windows-client/src-tauri/src/client/gui.rs

```diff
@@ -467,3 +467,25 @@ async fn run_controller(
     tracing::debug!("GUI controller task exiting cleanly");
     Ok(())
 }
+
+#[cfg(test)]
+mod tests {
+    #[test]
+    fn test_keyring() -> anyhow::Result<()> {
```

Comment:
Thanks for adding unit tests! Do these keys need to be cleaned up or do you assume that the VM will be fresh on each run?

Reply:
It wouldn't affect the outcome of our test, but yeah I went back and added code to clean them up. Otherwise it'll leave them in the Credential Manager indefinitely on dev systems:

![image](https://github.com/firezone/firezone/assets/13400041/845141bb-e645-4709-bf04-c055fc1ddebe)